pkg/storaged/btrfs/subvolume.jsx

```diff
@@ -206,6 +277,20 @@ export function make_btrfs_subvolume_page(parent, volume, subvol) {
         action: () => subvolume_create(volume, subvol, (mounted && !opt_ro) ? mount_point : mount_point_in_parent),
     });
 
+    let delete_excuse = "";
+    if (!mount_point_in_parent) {
+        delete_excuse = _("Parent subvolume needs to be mounted");
```

Comment:
Hmm, this needs to say "Parent subvolume needs to be mounted writable" if appropriate. This is not easy to get out of mount_point_in_parent... hmm...

Reply:
Also, can we say "at least one parent"?

Reply:
I like it!

Reply:
Ah, now it always says "writable".. clever, I guess that works.